tests/evm/test_chainid.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from zkevm_specs.evm import (
+    ExecutionState,
+    StepState,
+    verify_steps,
+    Tables,
+    RWTableTag,
+    RW,
+    Block,
+    Bytecode,
+    RWDictionary,
+)
+from zkevm_specs.util import rand_fq, RLC, U256
+from zkevm_specs.util.param import N_BYTES_WORD
+
+
+TESTING_DATA = (0x030201, rand_fq().n)
+
+
+@pytest.mark.parametrize("chainid", TESTING_DATA)
+def test_chainid(chainid: U256):
+    randomness = rand_fq()
+
+    block = Block(chainid=chainid)
+
+    bytecode = Bytecode().chainid()
+    bytecode_hash = RLC(bytecode.hash(), randomness)
+
+    tables = Tables(
+        block_table=set(block.table_assignments(randomness)),
+        tx_table=set(),
+        bytecode_table=set(bytecode.table_assignments(randomness)),
+        rw_table=set(
+            RWDictionary(9)
+            .stack_write(1, 1023, RLC(block.chainid.to_bytes(N_BYTES_WORD, "little"), randomness))
```

Comment:
If the init argument of RLC is `int`, it would convert it into little-endian with `n_bytes` (default to `32`), so we can save conversion from int to little-endian bytes a lot.
```suggestion
            .stack_write(1, 1023, RLC(block.chainid, randomness))
```